CHANGELOG.md

```diff
@@ -13,6 +13,11 @@ All notable changes to this project will be documented in this file.
   
 - Return a readable message when the server returns an array of errors instead of `Unknown error` ([#62](https://github.com/customerio/customerio-node/pull/62))
 
+## [3.0.0]
```

Comment:
Wasn't sure about the version here or if the "invite event" bit is necessary. (Probably not.)